packages/render-fidelity-tools/test/config.json

```diff
@@ -157,25 +137,206 @@
       }
```

Comment:
I would put the boom box toward the top, but we can leave the boom box with axes toward the bottom, since they are basically the same.